terraform/modules/ecs/vars.tf

```diff
@@ -137,6 +137,11 @@ variable "polytomic_resource_web_memory" {
   default     = 4096 // 4 GB
 }
 
+variable "polytomic_resource_web_storage" {
```

Comment:
Is there any reason to expose this for non-execution containers? I mean, I get that it ensures parity across containers, but this feels like a knob we don't need. Maybe we just expose it for syncs right now?

Reply:
That seems reasonable. Probably best to keep the options from growing unnecessarily.

Reply:
Exactly my thinking.